README.md

```diff
@@ -175,7 +175,8 @@ Useful tools for working with IPTV.
 
 Libraries and frameworks for working with IPTV data
 
-- [@iptv/xmltv](https://www.npmjs.com/package/@iptv/xmltv) - Fast xmltv parser and generator for Node and browser
+- [@iptv/xmltv](https://www.npmjs.com/package/@iptv/xmltv) - Fast xmltv parser and generator for Node and browsers
+- [@iptv/playlist](https://www.npmjs.com/package/@iptv/playlist) - Fast M3U parser and generator for Node and browsers
```

Comment:
Links are sorted by date added. Please move it to the bottom of the list.

Reply:
Done